src/components/CollectionHeader/CollectionHeader.css

```diff
@@ -0,0 +1,51 @@
+@import url('https://fonts.googleapis.com/css?family=Raleway&display=swap');
+
+@media only screen and (min-width: 600px){
+.collection-card{
+width: 375px;
+height: 200px;
+background-color: #FAE5DC;
+}
+.collect-header{
+line-height: 26px;   
+margin-top: 36px;   
+display: flex;
+width: 285px;
+height: 26px;
+flex-direction: row;
+justify-content: center;
```

Comment:
@mamady310 This looks really good! The title looks a little off center though, so work on adjusting that. Although you have the `justify-content` property set to center here, the `collect-header` includes the arrow icon. You want your title to be centered on the card so that it aligns with the quote. You're really close!